integration/test_graphql.py

```diff
@@ -89,6 +89,15 @@ def test_get_data(client):
             d_found = True
     assert a_found and d_found and len(objects) == 2
 
+def test_get_data_with_properties_default(client):
+    """Test GraphQL's Get clause with properties default (None)."""
+    result = client.query.get("Ship").with_limit(2).do()
+    objects = get_objects_from_result(result)
+    all_props_found = True
+    for obj in objects:
+        if not obj["name"] or not obj["size"] or not obj["description"]:
+            all_props_found = False
+    assert all_props_found and len(objects) == 2
```

Comment:
thanks for the test, but there are no references in the ship-schema. You could reuse the people schema?

Reply:
I have moved to using `Group` class which has a reference property to `Person` in the `people_schema`.